### DISCUSSION TITLE
Are the Backpacks balanced in the mid-late game?

### DISCUSSION THREAD

**Discussion by BeansGalaxy** [BeansGalaxy/Beans-Backpacks-2#55] [Feedback]
I see shulkers as mojang's implementation of modded backpacks. This mod is trying to cover what shulkers might've missed without making them obsolete. Personally I haven't played much survival with my mod to have a full ender chest of shulkers with every item in the game sorted like we typically do in vanilla survival.

**If once you reach the late game do you use the backpacks or are shulkers such a solve-all solution that it makes them useless?**

**Comment by Dimensions89** [comment]
- backpacks are nicer in my opinion because they can be put on the back so they take up less space in your inventory
- but if you want them to be more balanced maybe give them slightly more space like 2 more slots.
- also i think leather should be used in the recipes instaid of phantom membranes as that would make the mod more server friendly and iron backpacks are really early game too so atleast they use leather 

anyway just my opinion
feel free to contact me on discord if you have more questions!
Discord: dimensions89
P.S just found a bug. iron backpack recipie appears with gold backpack under golden backpack in JEI (i am on fabric but i dont remember if there is a forge version so just clairifying.)
![image](https://github.com/BeansGalaxy/Beans-Backpacks-2/assets/155260301/ad3da3a4-10e1-4de7-9554-b1e167efb275)

**Reply by BeansGalaxy** [reply]
I don't have a discord so I hope you'll see this here but the leather backpack is supposed to be the iron tier and Iron/Gold backpacks are supposed to be diamond tier if that makes sense. The buttons on the backpack more accurately describe the tier of them. Might raise the backpack size, I just recently changed netherite from 9 -> 11

Thats why metal backpacks use phantom membrane because it's not until you have a base and stop skipping the night when you get phantoms. Phantoms are just plain annoying + they have ZERO use in vanilla, but I can see how that would be hard/differently balanced on servers. Might look into changing phantom spawning rules but idk if it makes since for a backpack mod. 
If you know data-packs you can change the recipe's by overriding them. *([How to add Backpacks](https://github.com/BeansGalaxy/Beans-Backpacks-2/wiki/Custom-Backpacks))* Be aware that I'm planning on going over all of that to make simpler & more robust soon.

Finally the Iron & Gold Backpacks are technically the same item (`beansbackpacks:metal_backpack`) so I don't know how to seperate their recipes in either JEI or REI. Honestly I see it as a non issue and if it gets fixed it's because it's has either completly broke or fixed itself.
> Also It's cool to see you using Create with my mod since I made this mod ***specifically because*** my inventory would fill up with andesite casings, kelp, chutes, redstone and stuff I couldn't justify a toolbox for but never had the space.

**Reply by Dimensions89** [reply]
Yes create adds a lot of items so i was having troubles with storing andesite and other stuff and i love how the mod kind of felt like it goes with create as it kind of has the same style and atmosphere if you get what i mean. anyway thanks for the mod its perfect for create and other mods. Consider adding support for comforts mod that allows you to have an extra slot for storing a sleeping bag.

**Reply by BeansGalaxy** [reply]
Having a little rolled up sleeping bag on your backpack would look sick. I don't know how I would implement it though, would it be a new slot? if it's in your backpack do you automatically the bag down when you take the backpack off? sneak + hotkey to place a sleeping bag in the backpack? I'll add a note to my to-do to think abt it.

**Comment by Erinbjorn** [comment]
I haven't come across a particular issue during my playthroughs so far. It is more difficult to manage items than other backpack mods but thats the point. However I believe a very helpful change could be so that without placing the backpack down, you could take out not just the last stored item, but could choose from 3 or 4 last stored.

This wouldn't have much impact most of the time and wouldn't drastically change the originality of this mod, but would massively help when in a hectic and dangerous situations by allowing you to pick one of the few "top" items, as those will naturally be the ones you store last to have a quick access to. At the same time it could be used for building, crafting or any other stuff when your inventory is cramped

**Reply by BeansGalaxy** [reply]
This has come up a lot in development so I've had a lot of time to think about this and these are the reasons.
- Backpacks are meant to be used for item's you don't care about, like pumpkin seeds when you're exploring or fences and stairs when you're building. 
- Customizing your backpack wouldn't feel as important if you never saw it. That's why you have to place it.
- IRL you can take out things from the top of your backpack but you need to take if off if you need first thing you packed. 
- It's unique. 
- I've tried to keep these backpacks balanced just below a shulker since they solve the problem of bulk storage and backpacks solve the problem with clutter.

Btw if you're in hectic situations and using sprint a lot, I just recently added an option in the config to instantly place the backpack when you press your hot key. By enabling that and changing the key it's prevent you from accidentally placing it if that ever happened to you.

Also you said compared to other backpack mods, its more difficult to manage items but I hope it's not annoying. Does everything else feel responsive and intuitive?

**Reply by Erinbjorn** [reply]
Definitely not annoying, not by a long shot. Its just more difficult because its balanced. Most backpack mods let you have any amount of them in your inventory, while also having like 30 slots each once you get to the iron/gold upgrades, which I always disliked as it essentially made any other mod for means of item transport like Nifty carts obsolete. Your mod is just much more "realistic" in its behavior, and its definitely better that way, even if it a little harder to manage items.

The main point about taking out one of the first few items rather than just the top one goes in hand with my suggestion about armor. So far when I explored with your mod, I usually find a lot of tools and armor and since they take up 64 slots each, you cant put them in the backpack and so most of my inventory was packed with armor and tools I wanted to bring home, which made me put some important items into the backpack.

For example I was escaping a decent horde from a dungeon but ran out of blocks trying to block the way, and since there were more enemies around I couldnt place down the backpack to take out some more blocks or ropes to get me to surface. They were one of the top items but since I had no space in the inv I couldnt take each item out to get to them and thus nearly died - which brought me to my suggestion. I see your point and its absolutely fair, just wanna point out you can just swing your backpack under your arm IRL and get to the first few things packed without having to put it down, even while on the move.

That being said I still enjoy yours marginally more than other backpack mods
